### PR TITLE
Fix nodetool, cqlsh errors

### PR DESCRIPTION
**Pull request by sergeydobrodey**
Sorry, last commit was with errors. I've tested functionality against old repo and last PR was blind-merge to this repo. This time i checked it in k8s.
Dev container was tested with
kubectl run --rm -ti --image gcr.io/my_repo/cassandra:v12-dev cqlsh -- cqlsh cassandra

### CODE REVIEW

File: files/cassandra-env.sh [hunk 1]
```diff
@@ -0,0 +1,295 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (chrislovecnm) [inline files/cassandra-env.sh]: Is this somewhere in the distro?

**Review by chrislovecnm — COMMENTED** [review]
Did not push the button ... arg

**Comment by sergeydobrodey** [comment]
I'm not sure that properly understood your [comment](https://github.com/k8s-for-greeks/docker-cassandra-k8s/pull/4#discussion_r101057496), I've updated PR. Now cassandra-env.sh is copied from the C* distribution package.

**Comment by VinceMD** [comment]
I see DEV_CONTAINER . Is the thinking that only Dev environments should be able to use cqlsh but not a production one?

**Pull request merged**